eng/build.sh

```diff
@@ -17,47 +17,74 @@ scriptroot="$( cd -P "$( dirname "$source" )" && pwd )"
-  echo "  --sign                     Sign build outputs"
-  echo "  --publish                  Publish artifacts (e.g. symbols)"
-  echo "  --clean                    Clean the solution"
+  echo "  --build (-b)               Build all source projects."
+  echo "  --clean                    Clean the solution."
+  echo "  --pack                     Package build outputs into NuGet packages."
+  echo "  --publish                  Publish artifacts (e.g. symbols)."
+  echo "  --rebuild                  Rebuild all source projects."
+  echo "  --restore (-r)             Restore dependencies."
+  echo "  --sign                     Sign build outputs."
+  echo "  --test (-t)                Build and run tests."
   echo ""
 
   echo "Libraries settings:"
-  echo "  --framework                Build framework: net5.0 or net472 (short: -f)"
-  echo "  --coverage                 Collect code coverage when testing"
-  echo "  --testscope                Test scope, allowed values: innerloop, outerloop, all"
-  echo "  --testnobuild              Skip building tests when invoking -test"
-  echo "  --allconfigurations        Build packages for all build configurations"
+  echo "  --allconfigurations        Build packages for all build configurations."
+  echo "  --coverage                 Collect code coverage when testing."
+  echo "  --framework (-f)           Build framework: net5.0 or net472."
+  echo "  --testnobuild              Skip building tests when invoking -test."
+  echo "  --testscope                Test scope, allowed values: innerloop, outerloop, all."
   echo ""
 
   echo "Native build settings:"
-  echo "  --clang                    Optional argument to build using clang in PATH (default)"
-  echo "  --clangx.y                 Optional argument to build using clang version x.y"
+  echo "  --clang                    Optional argument to build using clang in PATH (default)."
+  echo "  --clangx.y                 Optional argument to build using clang version x.y."
   echo "  --cmakeargs                User-settable additional arguments passed to CMake."
-  echo "  --gcc                      Optional argument to build using gcc in PATH (default)"
-  echo "  --gccx.y                   Optional argument to build using gcc version x.y"
+  echo "  --gcc                      Optional argument to build using gcc in PATH (default)."
+  echo "  --gccx.y                   Optional argument to build using gcc version x.y."
+  echo ""
 
   echo "Command line arguments starting with '/p:' are passed through to MSBuild."
   echo "Arguments can also be passed in with a single hyphen."
+  echo ""
+
+  echo "Here are some quick examples:"
+  echo ""
+  echo "Build CoreCLR on Linux for x64 on release configuration:"
+  echo "./build.sh clr --os linux --arch x64 --configuration release"
+  echo ""
+  echo "Build Debug libraries with a Release runtime."
+  echo "./build.sh --subset clr+libs --os linux --arch x64 --runtimeConfiguration release"
+  echo ""
```

Comment:
How about adding `build -subset clr+libs -runtimeConfiguration Release` (it's what we suggest for library work in docs\workflow\building\libraries\README.md)

Reply:
And note that that one picks the same target platform you are building on

Reply:
FWIW the shortest form (that most of us use) is `build clr+libs -rc Release`.